src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts

```diff
@@ -1114,4 +1201,94 @@ export class NetworkAreaDiagramViewer {
             this.redrawVoltageLevelNode(otherNode, busNodeEdges, movedEdges);
         }
     }
+
+    private moveThreeWtEdge(
+        edge: SVGGraphicsElement,
+        edgeNode: SVGGraphicsElement,
+        offset: Point
+    ) {
+        const twtEdge: HTMLElement = <HTMLElement>edgeNode.firstElementChild;
+        if (twtEdge != null && twtEdge.tagName == 'polyline') {
+            const pointsAttributes = twtEdge.getAttribute('points');
+            if (pointsAttributes != null) {
+                const points = DiagramUtils.getPolylinePoints(pointsAttributes);
+                if (points != null) {
+                    // compute polyline points
+                    const edgeNodes = this.getEdgeNodes(edge);
+                    const threeWtMoved =
+                        edgeNodes[1]?.id == this.selectedElement?.id;
+                    const busNodeId1 = edge.getAttribute('busnode1');
+                    const unknownBusNode1 =
+                        busNodeId1 != null && busNodeId1.length == 0;
+                    const nodeRadius1 = this.getNodeRadius(
+                        busNodeId1 != null ? +busNodeId1 : -1
+                    );
+                    const edgeStart = DiagramUtils.getDistance(
+                        DiagramUtils.getPosition(edgeNodes[0]),
+                        DiagramUtils.getPosition(edgeNodes[1]),
+                        unknownBusNode1
+                            ? nodeRadius1[1] +
+                                  this.svgParameters.getUnknownBusNodeExtraRadius()
+                            : nodeRadius1[1]
+                    );
+                    const translation = new Point(
+                        offset.x - this.initialPosition.x,
+                        offset.y - this.initialPosition.y
+                    );
+                    const edgeEnd = threeWtMoved
+                        ? new Point(
+                              points[points.length - 1].x + translation.x,
+                              points[points.length - 1].y + translation.y
+                          )
+                        : points[points.length - 1];
+                    // move polyline
+                    const polylinePoints: string =
+                        edgeStart.x.toFixed(2) +
+                        ',' +
+                        edgeStart.y.toFixed(2) +
+                        ' ' +
+                        edgeEnd.x.toFixed(2) +
+                        ',' +
+                        edgeEnd.y.toFixed(2);
+                    twtEdge.setAttribute('points', polylinePoints);
+                    // move edge arrow and label
+                    if (edgeNode.children.length > 1) {
+                        this.moveEdgeArroWAndLabel(
```

Comment:
just realized the typo: 

```suggestion
                        this.moveEdgeArrowAndLabel(
```

Reply:
Fixed